### PR TITLE
Password restrictions done

### PR DESCRIPTION
**Pull request by lee-RoyGaseller**
Password must be between 5 and 12 characters and contain both letters and numbers.

### CODE REVIEW

File: src/main/java/codeu/controller/NewPasswordServlet.java [hunk 1]
```diff
@@ -41,17 +41,27 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
...
             return;
         }
+
         if (!newPassword.equals(reTypedNewPassword)) {
             request.setAttribute("error", "Passwords must match.");
             request.getRequestDispatcher("/WEB-INF/view/newPassword.jsp").forward(request, response);
             return;
         }
+
+        if(newPassword.length()<5||newPassword.length()>12 || !newPassword.matches("(.*[a-z].*)")|| !newPassword.matches( "(.*[0-9].*)")){
```
Comment (tucosh) [inline src/main/java/codeu/controller/NewPasswordServlet.java]: OK, but how about some tests ?

Comment (lee-RoyGaseller) [reply src/main/java/codeu/controller/NewPasswordServlet.java]: You mean in the NewPasswordServletTest?

Comment (tucosh) [reply src/main/java/codeu/controller/NewPasswordServlet.java]: Well, yes, this would definitely warrant a test in NewPasswordServletTest

**Comment by tucosh** [comment]
I tried this again and I still get the test failures.. do all the tests run ok for you?

**Pull request merged**